### PR TITLE
1299 Optional Phone

### PR DESCRIPTION
**Pull request by wbyoko**
Screenshot 2019-07-23 12 07 50

### CODE REVIEW

File: web-client/src/views/StartCase/ContactPrimary.jsx [hunk 1]
```diff
@@ -185,6 +185,14 @@ export const ContactPrimary = connect(
           >
             <label className="usa-label" htmlFor="phone">
               Phone Number
+              {contactsHelper.contactPrimary.phoneNumberLabelHint && (
+                <span>
+                  {' '}
+                  <span className="usa-hint">
```
Comment (rachaelparris) [inline web-client/src/views/StartCase/ContactPrimary.jsx:191]: Could you just make this a single span with a `margin-left-05` (or whatever works) on it? I'm guessing the extra span and the space are there for spacing?

**Review by rachaelparris — APPROVED** [review]

File: web-client/src/views/StartCase/ContactSecondary.jsx [hunk 2]
```diff
@@ -160,6 +160,14 @@ export const ContactSecondary = connect(
             >
               <label className="usa-label" htmlFor="secondaryPhone">
                 Phone Number
+                {contactsHelper.contactSecondary.phoneNumberLabelHint && (
+                  <span>
+                    {' '}
+                    <span className="usa-hint">
```
Comment (rachaelparris) [inline web-client/src/views/StartCase/ContactSecondary.jsx:166]: Same as above comment.

File: web-client/src/views/StartCase/ContactPrimary.jsx (hunk 1 above)
Comment (wbyoko) [reply web-client/src/views/StartCase/ContactPrimary.jsx:191]: we use `margin-left-05` instead of spaces?

Comment (rachaelparris) [reply web-client/src/views/StartCase/ContactPrimary.jsx:191]: I don't know that we do it either way as a rule. It's just an opinion, you don't have to change it. The span inside of a span is really what's throwing me off, but if it works that's fine.

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/StartCase/ContactPrimary.jsx (hunk 1 above)
Comment (wbyoko) [reply web-client/src/views/StartCase/ContactPrimary.jsx:191]: i'll switch them to fragments `<>...</>` instead of spans.

**Pull request merged**